src/components/ScreenshotSuccessDialog.tsx

```diff
@@ -0,0 +1,48 @@
+import { DialogTitle } from "@radix-ui/react-dialog";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+} from "./ui/dialog";
+import { Button } from "./ui/button";
+import { BugIcon } from "lucide-react";
+
+interface ScreenshotSuccessDialogProps {
+  isOpen: boolean;
+  onClose: () => void;
+  handleReportBug: () => Promise<void>;
+  isLoading: boolean;
+}
+
+export function ScreenshotSuccessDialog({
+  isOpen,
+  onClose,
+  handleReportBug,
+  isLoading,
+}: ScreenshotSuccessDialogProps) {
+  const handleSubmit = async () => {
+    await handleReportBug();
+    onClose();
+  };
+  return (
+    <Dialog open={isOpen} onOpenChange={onClose}>
+      <DialogContent>
+        <DialogHeader>
+          <DialogTitle>Screenshot captured and ready</DialogTitle>
+        </DialogHeader>
+        <DialogDescription className="">
```

Comment:
let's just remove this now (generally i find that users don't read things in small text so i try to avoid putting important things there)